src/lib/ServiceWatcher/ipfsStatus.ts

```diff
@@ -0,0 +1,49 @@
+import fetch from 'node-fetch'
+import { startStatusHandler, serviceState } from '../util/statusPoll'
+import env from '../../env'
+
+const { WATCHER_POLL_PERIOD_MS, WATCHER_TIMEOUT_MS, IPFS_HOST, IPFS_PORT } = env
+
+const versionURL = `http://${IPFS_HOST}:${IPFS_PORT}/api/v0/version`
+const peersURL = `http://${IPFS_HOST}:${IPFS_PORT}/api/v0/swarm/peers`
+
+const getStatus = async () => {
```

Comment:
Could move this function into the existing `ipfs.ts` class since the ipfs config is already there and it keeps all calls to ipfs in the same place. Then the service watcher constructor can include an instance of the `ipfs` class.

Same thing with `getStatus` for substrate, that could live in the `chainNode.ts` class and service watcher has an instance of `chainNode`.